### DISCUSSION TITLE
Platform agnostic feature.

### DISCUSSION THREAD

**Discussion by 4l1fe** [hyperjump-io/json-schema#10] [General]
> JSV is designed to run in a vanilla node.js environment, but has no dependencies on node.js specific libraries so it can be bundled for the browser.

Keep it up, man! That's what i'm looking for. I'm gonna use it on the backend and frotend sides.

**Comment by jdesrosiers** [comment]
Thanks! I also make an effort to ensure builds for the browser are as lightweight as possible.
